### ISSUE TITLE
Accessibility - ComboBox options are not read out correctly

### ISSUE THREAD

**Issue by melvsparks** [OfficeDev/office-ui-fabric-react#3434]
### Bug Report

- __Package version(s)__: Newest on fabric site
- __Browser and OS versions__: Chrome (Windows and Mac), IE 11, Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Maybe, Depends on extent of the issue

Requested priority: Blocking, needed for final product release of Outlook Calendar at the beginning of December

Products/sites affected: 
Outlook React Calendar

#### Describe the issue:
Narrator in edge or IE does not read out the comboBox options correctly when keying up and down through them. I would expect the same behavior as the dropdown menu.

#### Actual behavior:
All it reads out is the arrow key that has been typed. "Up arrow, down arrow"

#### Expected behavior:
If you turn Narrator on, then select the down  chevron button, then key through the options you would expect it to read out "option a one of ten selected, option b two of ten unselected, ...".

I would expect the same behavior that of the dropdown menu
https://developer.microsoft.com/en-us/fabric#/components/dropdown

### If applicable, please provide a codepen repro:
It can be repro'd on the official fabric page
https://developer.microsoft.com/en-us/fabric#/components/ComboBox

**Comment by mohitbagra** [comment]
+1. We'd also like to get this fixed.

**Comment by shdighe** [comment]
+1

**Comment by jspurlin** [comment]
Looking at the W3C example of an accessible comoboBox the current markup is correct (see: http://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html especially example 3)

**Comment by jspurlin** [comment]
The current behavior is expected and is "by design"
